Shipr-Helm/templates/shipr-frontend-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.frontend.name }}
+spec:
```

Comment:
labels, annotations?

Please read more here, if you are not familiar with why annotations are important. 
https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
Done